### ISSUE TITLE
file import-release-team-keyring missing after nodejs plugin installed macOS Monterey

### ISSUE THREAD

**Issue by dichensun88** [asdf-vm/asdf-nodejs#325]
macOS Monterey version 12.6.1
**asdf plugin-add nodejs 
bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring**
bash: /Users/xxxxxx/.asdf/plugins/nodejs/bin/import-release-team-keyring: No such file or directory

**Comment by augustobmoura** [comment]
`asdf-nodejs` doesn't use GPG signatures checksums anymore. After updating `asdf` and `asdf-nodejs` (or install it in a new machine) you can install new nodejs versions without any new extra steps.

You can update your asdf plugins by running:
```bash
asdf update
asdf plugin-update --all
```

**Comment by dichensun88** [comment]
Thank you ugusto Moura.
That file is still missing. I have tried to uninstall nodejs. Here is what I did.

% asdf plugin remove nodejs
% asdf plugin list nodes   
postgres
ruby
% asdf plugin-add nodejs
 % bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
bash: /Users/xxxx/.asdf/plugins/nodejs/bin/import-release-team-keyring: No such file or directory
% asdf update
HEAD is now at 7e7a1fa chore: release 0.10.2 (#1233)
Updated asdf to release v0.10.2
% asdf plugin-update --all
Updating nodejs to master
Already on 'master'
Your branch is up to date with 'origin/master'.
Updating ruby to master
Already on 'master'
Your branch is up to date with 'origin/master'.
Updating postgres to master
Already on 'master'
Your branch is up to date with 'origin/master'.
 % asdf plugin-add nodejs                                     
Plugin named nodejs already added
 % bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
bash: /Users/xxxxxx/.asdf/plugins/nodejs/bin/import-release-team-keyring: No such file or directory
